crates/subspace-networking/src/create.rs

```diff
@@ -107,8 +64,12 @@ pub struct Config {
     pub initial_random_query_interval: Duration,
     /// Defines a handler for the pieces-by-range protocol.
     pub pieces_by_range_request_handler: ExternalPiecesByRangeRequestHandler,
-    /// Defines relay mode and limit settings.
-    pub relay_config: RelayConfiguration,
+    /// Circuit relay server address.
+    ///
+    /// Example: /memory/<port>/p2p/<server_peer_id>/p2p-circuit
+    pub relay_server_address: Option<Multiaddr>,
+    /// Parent node instance (if any) to keep alive.
```

Comment:
Please, add more informative comment here why do we need a parent node to be kept alive like that. It's confusing now.